### PR TITLE
[TRANSLATION] Update Simplified Chinese translation [12/19].

### PR DESCRIPTION
**Pull request by njlyf2011**
## Purpose
The PR I submitted yesterday caused serious problems because of my personal reason. 
So I resubmit it now.

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-chinese

File: dll/win32/shell32/lang/zh-CN.rc [hunk 1]
```diff
@@ -262,23 +262,23 @@ BEGIN
...
-    LTEXT "地点:", 14008, 8, 75, 45, 10
+    LTEXT "地点：", 14008, 8, 75, 45, 10
     EDITTEXT 14009, 58, 75, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | ES_AUTOHSCROLL | NOT WS_TABSTOP
-    LTEXT "大小:", 14010, 8, 91, 45, 10
+    LTEXT "大小：", 14010, 8, 91, 45, 10
     EDITTEXT 14011, 58, 91, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
     LTEXT "在磁盘上的大小：", 140112, 8, 107, 55, 10
     EDITTEXT 14012, 58, 107, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
     LTEXT "", -1, 8, 123, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     LTEXT "创建：", 14014, 8, 131, 45, 10
```
Comment (eason329) [inline dll/win32/shell32/lang/zh-CN.rc:277]:
```suggestion
    LTEXT "创建日期：", 14014, 8, 131, 45, 10
```
As kernelbin suggested in last PR, I think it is better to translate this string in a unified form like the string below (`"创建日期："`, `"修改日期："`, and `"访问日期："`).

Also, I can't select and suggest line 245 which I think it is same as this due to limitation of GitHub, can you also change it?

File: dll/win32/shell32/lang/zh-CN.rc [hunk 2]
```diff
@@ -457,12 +457,12 @@ STYLE DS_SHELLFONT | WS_POPUP | WS_CAPTION
...
     CONTROL "", IDC_FILETYPES_LISTVIEW, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHAREIMAGELISTS | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 7, 20, 249, 115
     PUSHBUTTON "新建(&N)", IDC_FILETYPES_NEW, 140, 140, 55, 14, WS_TABSTOP
     PUSHBUTTON "删除(&D)", IDC_FILETYPES_DELETE, 200, 140, 55, 14, WS_TABSTOP
-    GROUPBOX "'%S' 扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
-    LTEXT "打开方式:", -1, 12, 180, 40, 10
+    GROUPBOX "“%S”扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
+    LTEXT "打开方式：", -1, 12, 180, 40, 10
     CONTROL "", IDC_FILETYPES_ICON, "STATIC", SS_BITMAP | SS_REALSIZEIMAGE, 85, 180, 10, 10
     LTEXT "程序名称", IDC_FILETYPES_APPNAME, 100, 180, 80, 10
     PUSHBUTTON "更改...(&C)", IDC_FILETYPES_CHANGE, 200, 180, 50, 14, WS_TABSTOP
```
Comment (eason329) [inline dll/win32/shell32/lang/zh-CN.rc:468]: In Windows, the ellipsis is usually place after the mnemonic (access key, 便捷鍵) in (Simplified) Chinese translation. 

Also, I just found that the order of this kind of strings is not (yet) unified in Simplified Chinese. So would you guys like to unify these strings to "XXX(&A)..." (eg: `更改(&C)...`) or "XXX...(&A)" (eg: `更改...(&C)`)?
> For reference, I decided to use "XXX(&A)..." format for Traditional Chinese

File: dll/win32/shell32/lang/zh-CN.rc [hunk 3]
```diff
@@ -714,15 +714,15 @@ CAPTION "自定义"
...
     AUTOCHECKBOX "将此模板应用到所有子文件夹(&S)", IDC_FOLDERCUST_CHECKBOX, 15, 50, 210, 15
     GROUPBOX "文件夹图片", IDC_STATIC, 5, 75, 230, 90, WS_TABSTOP
     LTEXT "对于缩略图视图，您可以在此文件夹上放置图片以提醒您内容。", IDC_STATIC, 15, 87, 115, 33
     PUSHBUTTON "更改图片(&P)...", IDC_FOLDERCUST_CHOOSE_PIC, 15, 125, 115, 15
     PUSHBUTTON "恢复默认(&R)", IDC_FOLDERCUST_RESTORE_DEFAULTS, 15, 144, 115, 15
-    LTEXT "预览:", IDC_STATIC, 139, 85, 81, 11
+    LTEXT "预览：", IDC_STATIC, 139, 85, 81, 11
     CONTROL "", IDC_FOLDERCUST_PREVIEW_BITMAP, "STATIC", SS_BITMAP | WS_GROUP, 150, 100, 60, 60
     GROUPBOX "文件夹图标", IDC_STATIC, 5, 170, 230, 65, WS_TABSTOP
     LTEXT "对于除缩略图以外的所有视图，您可以将标准""文件夹""图标更改为另一个图标。", IDC_STATIC, 15, 180, 210, 25
```
Comment (eason329) [inline dll/win32/shell32/lang/zh-CN.rc:728]:
```suggestion
    LTEXT "对于除缩略图以外的所有视图，您可以将标准“文件夹”图标更改为另一个图标。", IDC_STATIC, 15, 180, 210, 25
```

File: dll/win32/shell32/lang/zh-CN.rc [hunk 4]
```diff
@@ -803,33 +803,33 @@ BEGIN
...
+    IDS_DELETEMULTIPLE_TEXT "您确定要删除这“%1”项吗？"
+    IDS_DELETESELECTED_TEXT "您確定要删除所选的项目吗？"
+    IDS_TRASHITEM_TEXT "您确定想移动“%1”到回收站？"
+    IDS_TRASHFOLDER_TEXT "您确定想要将“%1”和它的所有内容发送到回收站吗？"
+    IDS_TRASHMULTIPLE_TEXT "您确定要将这“%1”项发送到回收站吗？"
+    IDS_CANTTRASH_TEXT "项目“%1”不能发送到回收站。您想要彻底删除它吗？"
+    IDS_OVERWRITEFILE_TEXT "此文件夹已包含名为“%1”的文件。\n\n您想要替换它吗？"
     IDS_OVERWRITEFILE_CAPTION "确认文件覆盖"
-    IDS_OVERWRITEFOLDER_TEXT "此文件夹已包含一个名为 '%1' 文件夹。\n\n如果目标文件夹中的文件具有相同的名称作为\n所选文件夹中的文件，他们将被替换。 你仍然想要移动或\n复制该文件夹吗？"
+    IDS_OVERWRITEFOLDER_TEXT "此文件夹已包含一个名为“%1”文件夹。\n\n如果目标文件夹中的文件具有相同的名称作为\n所选文件夹中的文件，他们将被替换。 你仍然想要移动或\n复制该文件夹吗？"
```
Comment (eason329) [inline dll/win32/shell32/lang/zh-CN.rc:832]:
```suggestion
    IDS_OVERWRITEFOLDER_TEXT "此文件夹已包含一个名为“%1”文件夹。\n\n如果目标文件夹中的文件具有相同的名称作为\n所选文件夹中的文件，他们将被替换。你仍然想要移动或\n复制该文件夹吗？"
```

File: dll/win32/shell32/lang/zh-CN.rc [hunk 5]
```diff
@@ -392,10 +392,10 @@ FONT 9, "宋体", 0, 0, 0x0
 BEGIN
     CONTROL "", 14000, "SysListView32", LVS_REPORT | LVS_SHOWSELALWAYS | LVS_SHAREIMAGELISTS | WS_BORDER | WS_TABSTOP, 10, 10, 220, 50
     GROUPBOX "选定位置的设置", -1, 10, 72, 220, 70
-    RADIOBUTTON "自定义大小(&C):", 14001, 20, 90, 80, 10, WS_TABSTOP
+    RADIOBUTTON "自定义大小(&C)：", 14001, 20, 90, 80, 10, WS_TABSTOP
     EDITTEXT 14002, 116, 103, 50, 14, WS_TABSTOP | ES_NUMBER
-    LTEXT "最大值(MB)(&A):", -1, 20, 105, 70, 10
-    RADIOBUTTON "不要将文件移动到回收站，而是彻底删除。(&R)", 14003, 20, 117, 170, 20, BS_MULTILINE | WS_TABSTOP
+    LTEXT "最大值(MB)(&A)：", -1, 20, 105, 70, 10
+    RADIOBUTTON "不将文件移动到回收站，而是彻底删除。(&R)", 14003, 20, 117, 170, 20, BS_MULTILINE | WS_TABSTOP
```
Comment (eason329) [inline dll/win32/shell32/lang/zh-CN.rc:398]: Actually I think the original translation is better. 😅

File: dll/win32/shell32/lang/zh-CN.rc [hunk 6]
```diff
@@ -457,12 +457,12 @@ STYLE DS_SHELLFONT | WS_POPUP | WS_CAPTION
...
 FONT 9, "宋体", 0, 0, 0x0
 BEGIN
-    LTEXT "已注册的文件类型(&T):", -1, 7, 10, 180, 10
+    LTEXT "已注册的文件类型(&T)：", -1, 7, 10, 180, 10
     CONTROL "", IDC_FILETYPES_LISTVIEW, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHAREIMAGELISTS | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 7, 20, 249, 115
     PUSHBUTTON "新建(&N)", IDC_FILETYPES_NEW, 140, 140, 55, 14, WS_TABSTOP
     PUSHBUTTON "删除(&D)", IDC_FILETYPES_DELETE, 200, 140, 55, 14, WS_TABSTOP
-    GROUPBOX "'%S' 扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
-    LTEXT "打开方式:", -1, 12, 180, 40, 10
+    GROUPBOX "“%S”扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
```
Comment (HBelusca) [inline dll/win32/shell32/lang/zh-CN.rc:464]: The quoted string shouldn't be space-separated with the rest of the string? 🤔

File: dll/win32/shell32/lang/zh-CN.rc [hunk 7]
```diff
@@ -472,18 +472,18 @@ END
...
     PUSHBUTTON "否(&N)", IDNO, 150, 122, 60, 14
     PUSHBUTTON "取消", IDCANCEL, 215, 122, 60, 14
     ICON 146, -1, 11, 10, 21, 20, SS_REALSIZECONTROL
-    LTEXT "此文件夹已包含名为 '%2' 的文件。", 12291, 44, 10, 231, 22, SS_NOPREFIX
-    LTEXT "此文件夹已包含一个名为 '%2' 的只读文件。", 12292, 41, 10, 222, 22, SS_NOPREFIX
-    LTEXT "此文件夹已包含一个名为 '%2' 的系统文件。", 12293, 41, 10, 222, 22, SS_NOPREFIX
-    LTEXT "您想要替换现有文件吗", -1, 44, 35, 228, 10, SS_NOPREFIX
+    LTEXT "此文件夹已包含名为“%2”的文件。", 12291, 44, 10, 231, 22, SS_NOPREFIX
+    LTEXT "此文件夹已包含一个名为“%2”的只读文件。", 12292, 41, 10, 222, 22, SS_NOPREFIX
+    LTEXT "此文件夹已包含一个名为“%2”的系统文件。", 12293, 41, 10, 222, 22, SS_NOPREFIX
```
Comment (HBelusca) [inline dll/win32/shell32/lang/zh-CN.rc:485]: Same question regarding space as above.

File: dll/win32/shell32/lang/zh-CN.rc (hunk 6 above)
Comment (eason329) [reply dll/win32/shell32/lang/zh-CN.rc:464]: I searched the web for example sentences, and I see Simplified Chinese users usually don't space-separated the quote.🤔

BTW,
```suggestion
    GROUPBOX "“%s”扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
```
"%s" is accidentally capitalised.

**Comment by njlyf2011** [comment]
I will improve them,thanks!

**Comment by njlyf2011** [comment]
@HBelusca :
I searched the translation about Microsoft Windows and found that there is no need to split quote.
Shell32_Win7
Win11 Folder Option

**Comment by njlyf2011** [comment]
I have resolved the errors raised above, except for the issue about Chinese quotation marks.

If you think it is necessary, I will solve the problem about quotation marks later.

**Comment by HBelusca** [comment]
> [...] except for problems about Chinese quotation marks.

As far as I can see from the screenshots, it seems Windows doesn't have any space after a quoted text/word and the next one(s).
If this is also what is being done in everyday-life in China then just keep it that way, I think...

**Comment by kernelbin** [comment]
> > [...] except for problems about Chinese quotation marks.
> 
> As far as I can see from the screenshots, it seems Windows doesn't have any space after a quoted text/word and the next one(s). If this is also what is being done in everyday-life in China then just keep it that way, I think...

In most fonts, full-width Chinese quotes `“”` are much wider than English quotes `""` , so there's no need to add additional space around it, and this is also the standard.

actually in daily life, many people did not follow a rule, It's more casual XD

File: base/setup/reactos/lang/zh-CN.rc [hunk 8]
```diff
@@ -129,7 +129,7 @@ END
```
Comment (eason329) [inline base/setup/reactos/lang/zh-CN.rc:129]: Uh there's some trivial problems I found **Above** this line but can't choose them due to GitHub limitation:

* Line 58: The access key should be capitalise but that line doesn't.
* The colon is half-width, maybe they can set to full-width?

**Comment by njlyf2011** [comment]
OK.
I hope can merge this PR as soon as possible.

File: base/setup/reactos/lang/zh-CN.rc [hunk 9]
```diff
@@ -96,21 +96,21 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
-    LTEXT "安装源:", IDC_STATIC, 18, 17, 74, 11
+    LTEXT "安装源：", IDC_STATIC, 18, 17, 74, 11
     EDITTEXT IDC_INSTALLSOURCE, 95, 16, 210, 13, ES_READONLY | ES_AUTOHSCROLL | WS_VISIBLE | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "架构:", IDC_STATIC, 18, 29, 74, 11
+    LTEXT "架构：", IDC_STATIC, 18, 29, 74, 11
     EDITTEXT IDC_ARCHITECTURE, 95, 28, 210, 13, ES_READONLY | ES_AUTOHSCROLL | WS_VISIBLE | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "计算机:", IDC_STATIC, 18, 41, 74, 11
+    LTEXT "计算机：", IDC_STATIC, 18, 41, 74, 11
     EDITTEXT IDC_COMPUTER, 95, 40, 210, 13, ES_READONLY | ES_AUTOHSCROLL | WS_VISIBLE | NOT WS_BORDER | NOT WS_TABSTOP
     LTEXT "显示器:", IDC_STATIC, 18, 53, 74, 11
```
Comment (eason329) [inline base/setup/reactos/lang/zh-CN.rc:107]:
```suggestion
    LTEXT "显示器：", IDC_STATIC, 18, 53, 74, 11
```

File: dll/win32/syssetup/lang/zh-CN.rc [hunk 10]
```diff
@@ -182,12 +182,12 @@ BEGIN
...
-    IDS_LAYOUTTEXT "Your default text input language and method is: %s keyboard layout"
+    IDS_LOCALETEXT "标准和格式设置为 %s，区域设置为 %s。"
+    IDS_LAYOUTTEXT "您的默认文本输入语言和方法是：%s 键盘布局"
     IDS_DATETIMETITLE "日期和时间"
     IDS_DATETIMESUBTITLE "为您的计算机配置正确的日期和时间。"
     IDS_PROCESSTITLE "注册组件"
-    IDS_PROCESSSUBTITLE "正在完成 ReactOS 最后安装阶段"
+    IDS_PROCESSSUBTITLE "正在完成 ReactOS 最后的安装阶段"
     IDS_THEMESELECTIONTITLE "外观"
     IDS_THEMESELECTIONSUBTITLE "选择您的主题:"
```
Comment (eason329) [inline dll/win32/syssetup/lang/zh-CN.rc:192]:
```suggestion
    IDS_THEMESELECTIONSUBTITLE "选择您的主题："
```

File: dll/win32/shell32/lang/zh-CN.rc [hunk 11]
```diff
@@ -1021,26 +1021,26 @@ BEGIN
     IDS_SPECIFY_ACTION "您必须指定一个操作。"
     IDS_INVALID_PROGRAM "找不到指定的程序。请确保文件名和路径正确无误。"
     IDS_REMOVE_ACTION "您确定要移除此操作吗？"
-    IDS_ACTION_EXISTS "已为此文件类型注册了操作 '%s' 。请输入其他名称，然后重试。"
+    IDS_ACTION_EXISTS "已为此文件类型注册了操作 “%s”。请输入其他名称，然后重试。"
```
Comment (eason329) [inline dll/win32/shell32/lang/zh-CN.rc:1024]:
```suggestion
    IDS_ACTION_EXISTS "已为此文件类型注册了操作“%s”。请输入其他名称，然后重试。"
```
Is the space redundant? 😅

File: dll/win32/shell32/lang/zh-CN.rc [hunk 12]
```diff
@@ -973,8 +973,8 @@ BEGIN
 
     IDS_SHLEXEC_NOASSOC "没有为 ReactOS 配置关联的程序以打开这种类型的文件。"
 
-    IDS_FILE_DETAILS "'%S' 扩展名的详细信息"
-    IDS_FILE_DETAILSADV "文件扩展名为 '%s' 的类型 '%s'。若要更改会影响所有的 '%s' 文件的设置，请单击高级。"
+    IDS_FILE_DETAILS "“%S”扩展名的详细信息"
```
Comment (eason329) [inline dll/win32/shell32/lang/zh-CN.rc:976]:
```suggestion
    IDS_FILE_DETAILS "“%s”扩展名的详细信息"
```

**Review by eason329 — APPROVED** [review]
LGTM now. Here is some other suggestions.

**Pull request merged**